### PR TITLE
Login work resumed

### LINKED ISSUE #675: [Login] Errors are displayed when you click the "Sign IN/UP" button.
**Date Time**
18-06-2021 11:34 AM

**Priority**
High

**Pre-conditions**
1. Download the 'development' repository
2. Run it on localhost

**Steps to Reproduce**
Go to l and click on "Sign IN/UP" button.

### PR DESCRIPTION
**Pull request by khvatskyi**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/EventsExpress/EventsExpress/projects/2)

## Code reviewers

- [ ] @iuriikhrystiuk

### Second Level Review

- [ ] @sand0

## Summary of issue

The "Sign IN/UP" button does not work

## Summary of change

Invalid variable calls fixed

## Testing approach

Go to https://eventsexpress-test.azurewebsites.net/ and check if Login works.

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EventsExpress/ClientApp/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
         "mdbreact": "4.19.0",
         "moment": "2.24.0",
         "moment-timezone": "0.5.26",
+        "oauth-sign": "0.9.0",
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/package.json]: Any reason for changing packages?

File: EventsExpress/ClientApp/src/containers/GoogleLogin.js [hunk 2]
```diff
@@ -6,37 +6,44 @@ import { withRouter } from 'react-router-dom';
...
+        this.props.loginGoogle(
+            response.tokenId,
+            response.profileObj.email,
+            response.profileObj.name,
+            response.profileObj.imageUrl
+        );
+    }
+
+    failedGoogle = response => {
+        console.log('Google auth failed')
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/containers/GoogleLogin.js]: Please remove console.log statements.

File: EventsExpress/ClientApp/src/containers/linked-auths/google-login-add-wrapper.js [hunk 3]
```diff
@@ -17,20 +18,25 @@ class GoogleLoginAdd extends Component {
         );
     }
 
+    failedGoogle = response => {
+        console.log('Google auth failed')
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/containers/linked-auths/google-login-add-wrapper.js]: Same here regarding console.log

File: EventsExpress/ClientApp/package.json (hunk 1 above)
Comment (khvatskyi) [reply EventsExpress/ClientApp/package.json]: I have added packages because of their use in the code by the path below
- containers/linked-auths/twitter-login-base.js
- components/login/login.js

Comment (iuriikhrystiuk) [reply EventsExpress/ClientApp/package.json]: I don't see anynew imports there. Are you sure?

File: EventsExpress/ClientApp/src/containers/linked-auths/google-login-add-wrapper.js [hunk 4]
```diff
@@ -7,7 +7,8 @@ import { setErrorAlert } from '../../actions/alert-action';
 import '../css/Auth.css';
 
 class GoogleLoginAdd extends Component {
-    googleResponseHandler = response => {
+
+    responseGoogle = response => {
```
Comment (sand0) [inline EventsExpress/ClientApp/src/containers/linked-auths/google-login-add-wrapper.js]: Why did you change the naming here?

File: EventsExpress/ClientApp/src/reducers/config.js [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
-import { GET_CONFIGS_PENDING, GET_CONFIGS_SUCCESS } from '../actions/config/get-configs-action'
+import { GET_CONFIG_PENDING, GET_CONFIG_SUCCESS } from '../actions/config/get-configs-action'
 
-export const reducer = (state = initialState.configs, action) => {
+export const reducer = (state = initialState.config, action) => {
     switch (action.type) {
-        case GET_CONFIGS_PENDING: return {
+        case GET_CONFIG_PENDING: return {
             ...state,
             isConfigsPending: false
```
Comment (sand0) [inline EventsExpress/ClientApp/src/reducers/config.js]: Do we use this state value anywhere? I guess no, cause as far as I see we set it here to the wrong 'false' instead of 'true', and never change. So, probably would be better to remove it generally

File: EventsExpress/ClientApp/package.json (hunk 1 above)
Comment (sand0) [reply EventsExpress/ClientApp/package.json]: My guessing is they are sub-dependencies of the twitter-login package, so we have not install them directly

**Review by iuriikhrystiuk — APPROVED** [review]
